### ISSUE TITLE
No Model Schema with JSON

### ISSUE THREAD

**Issue by sumang1983** [springfox/springfox#1480]
In this use case I am trying to generate documentation for Request Body. But it's not generating with the actual JSON.

Maven Depdencies:
```xml
		<dependency>
			<groupId>org.json</groupId>
			<artifactId>json</artifactId>
			<version>20160212</version>
		</dependency>
		<dependency>
			<groupId>io.springfox</groupId>
			<artifactId>springfox-swagger2</artifactId>
			<version>2.0.2</version>
		</dependency>
		<dependency>
			<groupId>cglib</groupId>
			<artifactId>cglib</artifactId>
			<version>2.2.2</version>
		</dependency>
		<dependency>
			<groupId>com.fasterxml.jackson.core</groupId>
			<artifactId>jackson-databind</artifactId>
			<version>${jackson.version}</version>
		</dependency>
		<dependency>
			<groupId>javax.xml.bind</groupId>
			<artifactId>jaxb-api</artifactId>
			<version>${jaxb-api.version}</version>
			<scope>runtime</scope>
		</dependency>
```
Controller Code :
```java
@RequestMapping(value = "/employee/add", method = RequestMethod.POST)
	@ApiOperation(consumes="application/json",value="Save Employee")
	public ResponseEntity<List<ObjectError>> saveEmployee(@ApiParam @RequestBody String requestBody,BindingResult bindingResult) {
		GsonBuilder gsonBuilder = new GsonBuilder();
		Gson gson = gsonBuilder.setDateFormat("yyyy-MM-dd").create();
			
                employee = gson.fromJson(requestBody, Employee.class);
		
		 return new ResponseEntity<List<ObjectError>>(new ArrayList<ObjectError>(), HttpStatus.OK);
	}
```
        
[APIController.txt.txt](https://github.com/springfox/springfox/files/456751/APIController.txt.txt)
[Employee.txt](https://github.com/springfox/springfox/files/456753/Employee.txt)
[HomeController.txt](https://github.com/springfox/springfox/files/456750/HomeController.txt)
[SwaggerConfig.txt](https://github.com/springfox/springfox/files/456752/SwaggerConfig.txt)

**Comment by dilipkrish** [comment]
I Would start by updating the springfox dependency to `2.5.0`

**Comment by sumang1983** [comment]
Any thing we need add some piece of code to get it fixed other than upgrading to 2.5.0? I can definitely try that..Also in this use case I am not using Object Mapper. From client request would be in JSON with the request body.
